crates/fluvio-cli/src/profile/add.rs

```diff
@@ -13,7 +12,8 @@ pub struct ManualAddOpt {
     cluster_address: String,
 
     /// Installation type of cluster, e.g. local, local-k8, k8
-    installation_type: Option<InstallationType>,
+    #[cfg(feature = "k8s")]
```

Comment:
why does `installation_type` need to be behind the `k8s` feature flag? shouldn't it be always available?

Reply:
@matheus-consoli This is my opinion, but in the big picture I can see this is reasonable for the local-first installation support we are working towards. This way we're not required to be tied to k8 on systems that might not support it.

The reason for this specific gate is because we include `fluvio-cluster` with the `k8s` feature, and we don't build windows fluvio cli with k8s support.

This is the failure we saw w/o this gate: https://github.com/infinyon/fluvio/actions/runs/6822041861/job/18553405148